Doc/library/inspect.rst

```diff
@@ -765,6 +765,18 @@ function.
        .. versionadded:: 3.10
           ``globalns`` and ``localns`` parameters.
 
+   .. classmethod:: Signature.from_code(co)
+
+       Return a :class:`Signature` (or its subclass) object
+       for a given :class:`code object <types.CodeType>` ``co``.
+
+       Since code objects do not know anything
+       about default values or annotations,
+       they will always be omitted from the signature.
+       It is recommended to use :meth:`Signature.from_callable`
+       when function object is available, it does not have these limitations.
```

Comment:
Thanks, it's more explicit and propose a concrete action to avoid these limitations!